assets/adr/catalog-import/examples.md

```diff
@@ -225,17 +225,17 @@ Response
 
 ```json
 {
-    "containsErrors": true,
-    "records": [
+  "containsErrors": true,
+  "records": [
```

Comment:
@jrson83 this file is imported. Sadly we can not change it, should be excluded.

https://github.com/shopware/shopware/tree/trunk/adr/assets

Could be adjusted in the source repo, but I don't know if smth fails on the source repo then.

Reply:
I reverted the file, is this ok?

Should be resolved by https://github.com/shopware/docs/pull/1596/commits/2b38c98adbec11309ce1e8329f9f7101fb0b7aa3

Reply:
There are some files left - best take a look at -> https://github.com/shopware/docs?tab=readme-ov-file#synced-files  or https://github.com/shopware/docs/blob/18ce51362740ab23816f9c09cc92b340f51d9414/.github/workflows/markdown-style-check.yml#L21

Reply:
Should be resolved by

https://github.com/shopware/docs/pull/1596/commits/7dfbd22e05a8b5b001e7d84289a2b3a256e2e479
https://github.com/shopware/docs/pull/1596/commits/24db25ec26811294c2c8b0757e755b7db03a03cf